.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: v8USzobOwk9BKjiXaih8UkJ2tevWWLxK5
```

Comment:
You can't have your token put in public due to security breach

Reply:
i removed the files but travis is not working any suggestion ??